src/nupic/encoders/ScalarEncoder.hpp

```diff
@@ -97,7 +97,7 @@ namespace nupic {
  * makes sense because, for example, with the input space [1, 10] and 10
  * buckets, 1.49 is in the first bucket and 1.51 is in the second.
  */
-class ScalarEncoder : public ScalarEncoderBase {
+class DllEXPORT ScalarEncoder : public ScalarEncoderBase {
```

Comment:
this should not be here?

Reply:
oops. missed on.  I thought I removed all of those that I had inserted.
I will get it.